src/app/comparison-portal/layout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Metadata } from 'next';
+
+import { Topbar } from '@/components/Topbar/Topbar';
+
+export const metadata: Metadata = {
+  title: 'Comparison Portal',
+};
+
+export default function Layout({ children }: { children: React.ReactNode }) {
+  return (
+    <div>
+      <Topbar />
+      <main className="flex flex-col gap-6 lg:gap-10 p-5 lg:p-10 text-content">{children}</main>
+    </div>
+  );
```

Comment:
Chatbot is added there in the topbar, but in a different way, should be the same here, see  #117